tests/accesscontrol/tests/access_control_security_context.go

```diff
@@ -87,6 +91,10 @@ var _ = Describe("Access-control security-context,", func() {
 
 	// 63738
 	It("two deployments, one pod each, one container each, both have allowed security context", func() {
+		if !tshelper.IsKindCluster() {
```

Comment:
I think the logic should remove the !

Reply:
:-) The logic and ginkgo are on opposing sides, "should do something, if it can":
 https://onsi.github.io/ginkgo/#skipping-specs

Reply:
I'm particularly concerned about executing the command multiple times. Is there a way to ensure it runs once without parallel execution? Alternatively, should we fully commit and utilize `sync.Once`?

Reply:
Not too worried about it.  We can switch to a singleton or something if performance is impacted later.

Reply:
Also I think you still have to remove the ! because if it's a Kind cluster, we want to skip.  Right now the way you have it makes me think if its not a kind cluster, skip.

Reply:
I've tested it, `!` is needed.